### ISSUE TITLE
swagger-inflector-json regression errors in 0.13.0

### ISSUE THREAD

**Issue by hirth-abi** [springwolf/springwolf-core#315]
**Describe the bug**

After updating a project from 0.11.1 to 0.13.0 we get several error messages during Spring Boot startup like 

```
ERROR i.s.o.i.examples.ExampleBuilder.fromProperty(451) - unable to convert `{country=string}` to JsonNode
ERROR i.s.o.i.examples.ExampleBuilder.fromProperty(451) - unable to convert `{id=string, name=string}` to JsonNode
```

The effect seems to be that those property objects are empty in the generated spec.

The same project was working fine in 0.11.1

**Dependencies and versions used**

Springwolf 0.13.0

**Code example**

Since this is an internal project, unfortunately I cannot provide an example project.

**Comment by timonback** [comment]
Hi @hirth-abi,
as commented in #314 I have trouble to replicate the issue.

Are you able to adapt the DTOs in i.e. https://github.com/springwolf/springwolf-core/blob/master/springwolf-examples/springwolf-kafka-example/src/main/java/io/github/stavshamir/springwolf/example/kafka/dtos/ExamplePayloadDto.java (in your fork) and then we have a look (together)?

**Comment by Brandom2725** [comment]
It's an issue with some wallets apparently or proof of your lost digital assets I had a similar issue. I contacted MATHEW LEWIS DEVELOPER directly and they had it resolved for me. 
mathewlouisdeveloper@gmail.com
Send them a mail or Contact them on Twitter @mathew_dev1 They'll help you out!

**Comment by timonback** [comment]
Closing due to inactivity and possible bugfix released in 0.14.0

Feel free to re-open if the issue persists with the current version of springwolf
